util/dvsim/Deploy.py

```diff
@@ -250,6 +250,18 @@ def set_status(self):
                     self.status = 'F'
                     break
 
+            # if returncode fails, usually encounter compile error, null pointer
```

Comment:
Perhaps this can be skipped if the fail patterns were found in the previous step?
```python
  if self.process.returncode != 0 and not self.fail_msg:
```

Reply:
Also on line 246, should we display 4 more lines following the error message (for cases line null pointer errors which will be indicated on subsequent lines)?
```python
                    (status, rslt) = subprocess.getstatusoutput(grep_cmd + " -A 4")
```

Reply:
agree, updated. Thanks